tcms/rpc/tests/test_testrun.py

```diff
@@ -489,20 +491,23 @@ def _fixture_setup(self):
         self.updated_build = BuildFactory()
         self.updated_summary = "Updated summary."
         self.updated_stop_date = datetime.strptime("2020-05-05", "%Y-%m-%d")
+        self.updated_planned_stop = datetime.strptime("2020-05-06", "%Y-%m-%d")
 
     def test_successful_update(self):
         update_fields = {
             "plan": self.updated_test_plan.pk,
             "build": self.updated_build.pk,
             "summary": self.updated_summary,
             "stop_date": self.updated_stop_date,
+            "planned_stop": self.updated_planned_stop,
```

Comment:
update both planned_start and planned_stop - this will exercise the custom logic I talked about above and by extension validate that you have redefined the 2 fields in the model form. You can use the 2 different input formats our custom DateTimeField supports.